src/registrar/models/domain.py

```diff
@@ -1666,6 +1670,51 @@ def _update_hosts_and_contacts(self, cleaned, fetch_hosts, fetch_contacts):
             if old_cache_contacts is not None:
                 cleaned["contacts"] = old_cache_contacts
 
+    def _update_hosts_and_ips_in_db(self, cleaned, fetch_hosts):
+        """Update hosts and host_ips in database if retrieved from registry.
+
+        Parameters:
+            self: the domain to be updated with hosts and ips from cleaned
+            cleaned: dict containing hosts.  Hosts are provided as a list of dicts, e.g.
+                [{"name": "ns1.example.com",}, {"name": "ns1.example.gov"}, "addrs": ["0.0.0.0"])]
+            fetch_hosts: boolean indicating whether or not fetch_hosts was called
+        """
+        if fetch_hosts:
```

Comment:
What do you think about putting this guy outside the method?

Reply:
+1 on this, but definitely non-blocking!

Reply:
I agree.  Done